install/kubernetes/cilium/files/spire/init.bash

```diff
@@ -0,0 +1,53 @@
+# shellcheck disable=SC2086
+# shellcheck disable=SC2139
+set -e
+
+echo "Waiting for spire process to start"
+while ! pgrep spire-server > /dev/null; do sleep 5; done
+
+SPIRE_SERVER_ROOT_PATH="/proc/$(pgrep spire-server)/root"
+
+alias spire_server="${SPIRE_SERVER_ROOT_PATH}/opt/spire/bin/spire-server"
+SOCKET_PATH="${SPIRE_SERVER_ROOT_PATH}/tmp/spire-server/private/api.sock"
+SOCKET_FLAG="-socketPath ${SOCKET_PATH}"
+
+echo "Checking spire-server status"
+while ! spire_server entry show ${SOCKET_FLAG} &> /dev/null; do
+  echo "Waiting for spire-server to start..."
+  sleep 5
+done
+
+echo "Spire Server is up, initializing cilium spire entries..."
+
+AGENT_SPIFFE_ID="spiffe://{{ .Values.auth.mTLS.spire.trustDomain }}/ns/{{ .Values.auth.mTLS.spire.install.namespace }}/sa/spire-agent"
+AGENT_SELECTORS="-selector k8s_psat:agent_ns:{{ .Values.auth.mTLS.spire.install.namespace }} -selector k8s_psat:agent_sa:spire-agent"
+CILIUM_AGENT_SPIFFE_ID="spiffe://{{ .Values.auth.mTLS.spire.trustDomain }}/{{ .Values.auth.mTLS.spire.identities.ciliumAgent }}"
+CILIUM_AGENT_SELECTORS="-selector k8s:ns:{{ .Release.Namespace }} -selector k8s:sa:{{ .Values.serviceAccounts.cilium.name }}"
+DCLIENT_SPIFFE_ID="spiffe://{{ .Values.auth.mTLS.spire.trustDomain }}/{{ .Values.auth.mTLS.spire.identities.delegatedClient }}"
+CILIUM_OPERATOR_SPIFFE_ID="spiffe://{{ .Values.auth.mTLS.spire.trustDomain }}/{{ .Values.auth.mTLS.spire.identities.ciliumOperator }}"
+CILIUM_OPERATOR_SELECTORS="-selector k8s:ns:{{ .Release.Namespace }} -selector k8s:sa:{{ .Values.serviceAccounts.operator.name }}"
+DCLIENT_SPIFFE_ID="spiffe://{{ .Values.auth.mTLS.spire.trustDomain }}/{{ .Values.auth.mTLS.spire.identities.delegatedClient }}"
+
+while pgrep spire-server > /dev/null;
+do
```

Comment:
Are we going to need to support changing these values after initial install?

Reply:
I see we can't do it in a declarative way either unfortunately. There is an open issue for it though: https://github.com/spiffe/spire/issues/2134